src/views/Components/PlayerSettings.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState } from 'react';
+import Button from '../../elements/Button'
+import arrowButton from '../../assets/svg-icons/arrow.svg';
+
+function Settings() {
+    function createPlayers() {
+        return [
+            {
+                id: 0,
+                name: "Player 1",
+            },
+            {
+                id: 1,
+                name: "Player 2",
+            },
+            {
+                id: 2,
+                name: "Player 3",
+            },
+            {
+                id: 3,
+                name: "Player 4",
+            },
+        ];
+    }
+    const [playerNames, setPlayerNames] = useState(createPlayers());
+    const [idForPlayerName, setIdForPlayerName] = useState(5);
```

Comment:
I think we should drop this state value and just look at the ids of the players, get the max id, and increment by 1 in there. Thoughts?